python/cugraph/cugraph/dask/structure/mg_property_graph.py

```diff
@@ -1204,7 +1224,9 @@ def edge_props_to_graph(
 
         # If a default_edge_weight was specified but an edge_weight_property
         # was not, a new edge weight column must be added.
-        elif default_edge_weight:
+        elif default_edge_weight or create_with_edge_info:
```

Comment:
The comment above looks like it needs to be updated now.

Reply:
undid this change